autoload/lsp/handlers.vim

```diff
@@ -41,8 +41,13 @@ def ProcessShowMsgNotif(lspserver: dict<any>, reply: dict<any>)
     # them.
     return
   endif
-  var mtype = LspMsgTypeToString(reply.params.type)
-  :echomsg $'Lsp({lspserver.name}):[{mtype}]: {reply.params.message}'
+  if reply.params.type == 1
+    util.ErrMsg($'Lsp({lspserver.name}) {reply.params.message}')
+  elseif reply.params.type == 2
+    util.WarnMsg($'Lsp({lspserver.name}) {reply.params.message}')
+  elseif reply.params.type == 3
+    util.InfoMsg($'Lsp({lspserver.name}) {reply.params.message}')
+  endif
```

Comment:
The specifications states that there is a type = 4, called hint. I think you should add that as well, as well a fallback if type > 4

Reply:
the specification said it only had 4 level, but ok let me added a safeguard as well.